config/config.exs

```diff
@@ -7,7 +7,7 @@ config :logger,
   handle_sasl_reports: false
 
 config :logger, :console,
-  format: {MongoosePush.LoggerFmt, :format},
+  format: {MongoosePush.Logger.Fmt, :format},
```

Comment:
Technically the format is called: `logfmt`, not `fmt`. I would suggest to change this name throughout to be more accurate - `fmt` alone feels weird :)